.github/workflows/e2e_tests.yml

```diff
@@ -73,9 +81,14 @@ jobs:
 
       - name: set up go 1.15
         uses: actions/setup-go@v1
+        if: ${{ !contains(github.event.ref, 'fips') }}
```

Comment:
I don't think we need to run the test framework itself using boring go. Here and for other tests.